hs-ucan/library/Web/UCAN/Internal/Orphanage/RSA2048/Public.hs

```diff
@@ -35,15 +36,14 @@ instance Arbitrary RSA.PublicKey where
     return . fst . Unsafe.unsafePerformIO $ RSA.generate 2048 exp
 
 instance Display RSA.PublicKey where
-  textDisplay pk =
-    X509.PubKeyRSA pk
+  display pk =
+    pk
+      & X509.PubKeyRSA
       & X509.pubKeyToPEM
-      & PEM.pemWriteBS
-      & decodeUtf8Lenient
-      & Text.strip
-      & Text.dropPrefix pemHeader
-      & Text.dropSuffix pemFooter
-      & Text.filter (/= '\n')
+      & PEM.pemContent
+      & BS64.decodeLenient
+      & Builder.byteString
+      & Utf8Builder
```

Comment:
Oh yeah, that's way better 🎉

Reply:
Haha, you're complementing your own code in this case though! :P (I've copied this over from the other PR)